universal-application-tool-0.0.1/test/support/TestQuestionBank.java

```diff
@@ -141,6 +226,10 @@ private static Question maybeSave(QuestionDefinition questionDefinition) {
     APPLICANT_NAME,
     APPLICANT_ADDRESS,
     APPLICANT_FAVORITE_COLOR,
-    APPLICANT_HOUSEHOLD_MEMBERS
+    APPLICANT_HOUSEHOLD_MEMBERS,
```

Comment:
(blocking): there may be a better way to sort this if the list grows really long, but for now lets make it alphabetical. and the methods that return the questions should be ordered in a similar way.